.github/workflows/TSlint.yml

```diff
@@ -0,0 +1,27 @@
+name: TS Lint
+on: [push]
+
+jobs:
+  lint:
+    name: Lint Typescript
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
I recommend upgrading to Node 14, as 12 is about to enter the final maintenance mode: https://nodejs.org/en/about/releases/